install/v2plugin/README.md

```diff
@@ -0,0 +1,61 @@
+# Docker 1.13/17.03 v2plugin support
+
+Docker 1.13/17.03 supports legacy plugins (binaries/containers) and docker managed plugins (v2plugin) using docker plugin commands. Contiv binaries (netplugin and netmaster) and Contiv container (contiv/netplugin) support legacy plugin mode. In addition, Contiv can be run as v2plugin (contiv/v2plugin).
+
+## Classic Swarm and Swarm-mode
+In Classic Swarm or Legacy Docker Swarm, swarm binary/container running on each host forms a cluster using an external key-value store. Network plugins can run in the legacy mode. Docker versions upto 1.12 supported only this mode for remote drivers. In this mode docker engine is started with a cluster-store option. Docker 1.13.1 also supports this mode and the plugin can be run legacy plugin or v2plugin. Contiv binaries/containers will be supported in this legacy mode. 
+
+Docker introduced swarm-mode in 1.12 where the docker engines form a cluster without using an external key-value store. Swarm-mode in Docker 1.12 only supported docker overlay network driver. In this mode the swarm is initialized using docker swarm commands. From Docker 1.13.1, remote network drivers implemented as v2plugins are also supported in swarm-mode. Contiv v2plugin support for docker swarm-mode is still in progress.
+
+## v2plugin 
+Docker managed plugins (https://docs.docker.com/engine/extend/) are run as runc containers and are managed using docker plugin commands. Docker engine running in swarm-mode (https://docs.docker.com/engine/swarm/) requires the remote drivers to implement v2plugin architecture.
+
+## Contiv plugin install
+### docker store
+Docker certified contiv plugin is avaliable on Docker Store (https://store.docker.com/plugins/803eecee-0780-401a-a454-e9523ccf86b3?tab=description).
+```
+docker plugin install store/contiv/v2plugin:1.0.0-beta.3 iflist=eth1,eth2
```

Comment:
Does this command take any other netplugin specific args? If so, should we document them(not necessarily in this document) or does installer take care of this?